### ISSUE TITLE
Missing tutorial for installation

### ISSUE THREAD

**Issue by asterismo** [froodyapp/froody-server#1]
Can you summarize the steps to install it on a Debian 8 Jessie Server? I created the nginx vhost, and installed dependencies with composer, what else?

**Comment by gsantner** [comment]
I'm using apache, but it shoudln't be too difficult to set it up on nginv too. All needs to be done is to allow froody to override paths. 

This is my apache config:
```
Alias /froody /var/www/froody/
<Directory "/var/www/froody">
	ErrorDocument 404 default
	Options FollowSymLinks
	AllowOverride All
</Directory>
```

As you noticed, yes you need to install dependencies via composer first.

For the database: Standard mariadb/mysql installation, create database according to sql schema and set credentials in froody-servers's config.php.

Summed up: Pretty standard installation, nothing special
